### ISSUE TITLE
Stopped working in WP 5.7

### ISSUE THREAD

**Issue by adriantoll** [joppuyo/remove-drop-cap#2]
I'm using the nightly 5.7 build to test some other things, and unfortunately that version stops this plugin from working. Not sure why exactly - I'll be doing some digging to see if there's another way of doing it - but thought it worth flagging in advance.

**Comment by joppuyo** [comment]
Hey,

Thanks for the info, it's so nice to play cat and mouse with Gutenberg developers every release 😕 It looks like the typography was moved from `global` to `defaults` so the updated code would be:

```php
add_filter(
    'block_editor_settings',
    function ($editor_settings) {
        $editor_settings['__experimentalFeatures']['defaults']['typography']['dropCap'] = false;
        return $editor_settings;
    }
);

```

I'm going to wait until the final 5.7 release to update this plugin. Who knows if they still manage to change something before release.

Thanks!

**Comment by adriantoll** [comment]
@joppuyo I know what you mean about the cat-and-mouse. :roll_eyes: Thanks for the pointer - I can confirm that works in 5.7 nightly at least.

**Comment by joppuyo** [comment]
Fixed in v1.2.0!
